homework/style.css

```diff
@@ -1,3 +1,191 @@
 .alert-error {
```

Comment:
Your CSS still needs some tweaking. When I start your application I get this:

![image](https://user-images.githubusercontent.com/2788771/62775645-87f89c80-baa8-11e9-91cf-45ae56fdcda9.png)

When I switch to mobile emulation in Chrome Developer tools I get this:

![image](https://user-images.githubusercontent.com/2788771/62775692-aa8ab580-baa8-11e9-9ac8-625cd1db2e23.png)

When I switch back again to non-mobile, the number of contributions seems to be in the right place. Looks like some timing / animation issue.

![image](https://user-images.githubusercontent.com/2788771/62775753-da39bd80-baa8-11e9-928d-937bf185a5de.png)